Formula/dropbear.rb

```diff
@@ -40,6 +40,6 @@ def install
     testfile = testpath/"testec521"
     system "#{bin}/dbclient", "-h"
     system "#{bin}/dropbearkey", "-t", "ecdsa", "-f", testfile, "-s", "521"
-    assert testfile.exist?
+    assert_predicate testpath/testfile, :exist?, "Test Failed"
```

Comment:
`testfile` already contains `testpath`… so I'm not sure why this is passing?
Also, there is a single test so the (uninformative) error message is not needed.